app/views/layouts/application.html.erb

```diff
@@ -52,7 +52,7 @@
       window.agentPaths = {};
       window.agentNames = [];
       <% if current_user.present? -%>
-        var myAgents = <%= Utils.jsonify(current_user.agents.pluck(:name, :id).inject({}) {|m, a| m[a.first] = agent_path(a.last); m }) %>;
+        var myAgents = <%= Utils.jsonify(current_user.agents.pluck(:name, :id).inject({}) {|m, a| next if a.last.nil?; m[a.first] = agent_path(a.last); m }) %>;
```

Comment:
Took me some bisecting, with https://github.com/rails/rails/pull/25976 `pluck` now doesn't always query the database but uses the AR association if it is already loaded. We are doing `current_user.agents.build` in the controller which means that `pluck` now also returns `[nil, nil]` for the new record. I think that is a bug (at least for a patch release), but I'll have to wait for some feedback from the rails team.